uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_contact-actions-menu.scss

```diff
@@ -0,0 +1,61 @@
+.crm_contact-actions {
```

Comment:
Did you only change the file name, or the content also?

Reply:
Both filename and content is changed!